tests/nnfw_api/src/one_op_tests/Add.cc

```diff
@@ -122,10 +122,10 @@ TEST_F(GenModelTest, neg_OneOp_Add_InvalidShapeConst)
   std::vector<float> rhs_data{5, 4, 0, 7, 4, 0};
   uint32_t rhs_buf = cgen.addBuffer(rhs_data);
   int lhs = cgen.addTensor({{1, 2, 2, 1}, circle::TensorType::TensorType_FLOAT32});
-  int rhs = cgen.addTensor({{1, 2, 2, 1}, circle::TensorType::TensorType_FLOAT32, rhs_buf});
+  int rhs = cgen.addTensor({{1, 2, 3, 1}, circle::TensorType::TensorType_FLOAT32, rhs_buf});
   int out = cgen.addTensor({{1, 2, 3, 1}, circle::TensorType::TensorType_FLOAT32});
   cgen.addOperatorAdd({{lhs, rhs}, {out}}, circle::ActivationFunctionType_NONE);
-  cgen.setInputsAndOutputs({lhs, rhs}, {out});
+  cgen.setInputsAndOutputs({lhs}, {out});
```

Comment:
Not directly related to this commit, but introducing `InputOutputChecker` found a bug in the test case.

- `rhs` is a const so it shouldn't be an input
- If rhs and lhs shapes match, the current shape inference do not throw (it does not try to match output shape, just calculates new output shape)